tests/test_checker.py

```
@@ -0,0 +1,355 @@
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)